README.md

```diff
@@ -19,76 +19,72 @@
+which helps it perform just as well as `mtl`'s! Add `polysemy-plugin` to your
+`package.yaml` or `.cabal` file's `dependencies` section to use. Then turn it on with a pragma in your source files:
 
 ```haskell
 {-# OPTIONS_GHC -fplugin=Polysemy.Plugin #-}
 ```
-Or by adding `-fplugin=Polysemy.Plugin` to your package.yaml/.cabal file `ghc-options` section.
 
+Or by adding `-fplugin=Polysemy.Plugin` to your `package.yaml`/`.cabal` file `ghc-options` section.
 
 ## Features
 
-* *Effects are higher-order,* meaning it's trivial to write `bracket` and `local`
+- *Effects are higher-order,* meaning it's trivial to write `bracket` and `local`
     as first-class effects.
-* *Effects are low-boilerplate,* meaning you can create new effects in a
+- *Effects are low-boilerplate,* meaning you can create new effects in a
     single-digit number of lines. New interpreters are nothing but functions and
     pattern matching.
-* *Effects are zero-cost,* meaning that GHC<sup>[1](#fn1)</sup> can optimize
-    away the entire abstraction at compile time.
-
-
-<sup><a name="fn1">1</a></sup>: Unfortunately this is not true in GHC 8.6.3, but
-will be true in GHC 8.10.1.
-
 
 ## Tutorials and Resources
 
-- Raghu Kaippully wrote a beginner friendly [tutorial](https://haskell-explained.gitlab.io/blog/posts/2019/07/28/polysemy-is-cool-part-1/index.html).
-- Paweł Szulc gave a [great talk](https://youtu.be/idU7GdlfP9Q?t=1394) on how to start thinking about polysemy.
-- I've given a talk on some of the [performance implementation](https://www.youtube.com/watch?v=-dHFOjcK6pA)
-- I've also written [some](http://reasonablypolymorphic.com/blog/freer-higher-order-effects/) [blog posts](http://reasonablypolymorphic.com/blog/tactics/) on other implementation details.
-
+- Raghu Kaippully wrote a beginner friendly
+  [tutorial](https://haskell-explained.gitlab.io/blog/posts/2019/07/28/polysemy-is-cool-part-1/index.html).
+- Paweł Szulc gave a [great talk](https://youtu.be/idU7GdlfP9Q?t=1394) on how
+  to start thinking about polysemy.
+- Sandy Maguire, the author, gave a talk on some of the
+  [performance implementation](https://www.youtube.com/watch?v=-dHFOjcK6pA)
+- He has also written
+  [some](http://reasonablypolymorphic.com/blog/freer-higher-order-effects/)
+  [blog posts](http://reasonablypolymorphic.com/blog/tactics/) on other
+  implementation details.
 
 ## Examples
 
-Make sure you read the [Necessary Language
-Extensions](https://github.com/isovector/polysemy#necessary-language-extensions)
+Make sure you read the
+[Necessary Language Extensions](https://github.com/polysemy-research/polysemy#necessary-language-extensions)
 before trying these yourself!
 
 Teletype effect:
 
 ```haskell
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE LambdaCase, BlockArguments #-}
-{-# LANGUAGE GADTs, FlexibleContexts, TypeOperators, DataKinds, PolyKinds #-}
+{-# LANGUAGE TemplateHaskell, LambdaCase, BlockArguments, GADTs
```

Comment:
this example isn't very helpful, really

Reply:
Hmm, maybe it's good to have "dumb" example first, not doing much more than interpreting simple effect, as a sort of showcase of `mkSem` and `interpret`.

Reply:
Should I keep it as it is?